### ISSUE TITLE
Some ISOs show up as Audio CD 1

### ISSUE THREAD

**Issue by BiggerJ** [mihaip/infinite-mac#103]
This has happened to me with ~~Maddness of Roland and~~ the Japanese Wacky Races game.

**Comment by mihaip** [comment]
Is the native version of Basilisk II or SheepShaver able to mount them? Do you have a link to the specific ISOs that you tried, so that I can make sure I'm reproducing the issue?

**Comment by bshep** [comment]
If I had to guess its a Hybrid ISO with HFS and ISO9660 tracks, i've had issues in the past where this happens, IIRC i ended up extracting the track and just mounting that, a link to the ISO would be helpful as mihaip mentioned.

**Comment by BiggerJ** [comment]
Madness of Roland: https://archive.org/download/madnessofroland/MADNESS%20of%20Roland.iso

Wacky Races: https://archive.org/download/macwackyraces/%28MAC%29%20Wacky%20Races.zip

**Comment by that-ben** [comment]
BIN/CUE contains multiple tracks, usually 1 or more audio tracks and one DATA track (files).  That may be why it's mounting an audio CD.  You can convert a BIN/CUE DATA track to an ISO disk image (dropping all audio tracks) so that at least the game/app works in SheepShaver: https://www.macintoshrepository.org/articles/37-i-downloaded-something-that-contains-bin-cue-files-how-do-i-mount-this-under-mac-os-9-

**Comment by BiggerJ** [comment]
The version of Wacky Races I tried already was an iso (I'd initially linked to a copy that was a bin/cue by mistake).

**Comment by that-ben** [comment]
OK.  Well then, I must admit that I also experienced that weird issue where only an AUDIO CD was mounted on the desktop, even though I was trying different disk images, restarting the emulator, reloading the page, closing the tab, etc... At some point I had to clear my cache for the domain macos9.app and then the issue disappeared.  Reference: https://github.com/mihaip/infinite-mac/issues/85#issuecomment-1426845419

I suggest that you try to clear your cache in your browser and load macos9.app from scratch.

**Comment by BiggerJ** [comment]
My mistake - I've struck it out in my posts. I'm redownloading the Wacky Races ISO to make sure it's the same one I tried.

**Comment by BiggerJ** [comment]
I've just downloaded the Wacky Races ISO I linked, and unlike the one I tried, this one actually works. I think it might have been [this one](https://archive.org/details/wackyraces_201812). Sorry for all the mess.

**Comment by BiggerJ** [comment]
I just discovered the alternate Wacky Races image I found was a .img image, not a .iso image (I was in a hurry and didn't realize). It has exactly the same filesize as the copy I had trouble with, and changing its dilename or .iso or .dsk doesn't help. I don't know what its deal is and it might be a special case. What bshep said about hybrid disk images might be something to look into - I'll go look for a hybrid disc image to check.
